client/components/domains/domain-search-results/index.jsx

```diff
@@ -67,7 +81,7 @@ var DomainSearchResults = React.createClass( {
 		} else if ( this.props.suggestions && this.props.suggestions.length !== 0 && this.isDomainUnavailable() ) {
 			if ( this.props.products.domain_map && this.props.lastDomainError.code === 'not_available_but_mappable' ) {
 				const components = { a: <a href="#" onClick={ this.addMappingAndRedirect }/>, small: <small /> };
-				if ( domainsWithPlansOnlyTestEnabled ) {
+				if ( ! this.props.plansOnlyOldPricing ) {
```

Comment:
I think you meant `this.props.domainsWithPlansOnly` here - `plansOnlyOldPricing` is never actually set in the code anywhere. This results in showing the message `
If you purchased cnn.com elsewhere, you can map it with WordPress.com Premium.` to non-DWPO users when searching for a mappable domain.